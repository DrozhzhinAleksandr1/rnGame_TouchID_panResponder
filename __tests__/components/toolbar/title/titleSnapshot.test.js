import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { Title } from '/components/toolbar/title/title';
import { localization } from '../../../../app/common/localization';


it('create snapshot title', () => {
    const renderer = new ShallowRenderer()
    let TitleSnapshot = renderer.render(<Title CurrentLanguage={localization.ru} />);
    expect(TitleSnapshot).toMatchSnapshot();
});