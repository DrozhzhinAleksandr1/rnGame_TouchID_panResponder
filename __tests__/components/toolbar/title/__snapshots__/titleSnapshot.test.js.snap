// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot title 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessibilityLabel="TabTitleID"
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Regular",
        "fontSize": 20,
      }
    }
    testID="TabTitleID"
  >
    Валюты
  </Text>
</Component>
`;
